nca/CoreDS/ConnectionSet.py

```diff
@@ -550,3 +570,114 @@ def get_non_tcp_connections():
         res.add_all_connections([ProtocolNameResolver.get_protocol_number('TCP')])
         return res
         # return ConnectionSet(True) - ConnectionSet.get_all_TCP_connections()
+
+    # TODO - after moving to the optimized HC set implementation,
+    #  get rid of ConnectionSet and move the code below to ConnectivityProperties.py
+    @staticmethod
+    def tcp_properties_to_fw_rules(tcp_props, cluster_info, peer_container, ip_blocks_mask,
+                                   connectivity_restriction):
+        """
```

Comment:
update the renaming of `tcp_props` to `conn_props`

Reply:
Updated